src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -581,7 +581,12 @@ private void ProcessPingFrame(FrameHeader frameHeader)
 
             // Send PING ACK
             // Don't wait for completion, which could happen asynchronously.
-            LogExceptions(SendPingAckAsync(_incomingBuffer.ActiveMemory.Slice(0, FrameHeader.PingLength)));
+            // We discard the the frame from the incoming buffer, possibly before
```

Comment:
Not sure what this comment means?

Reply:
Generally speaking, all these routines read the frame completely before returning. So I don't think you need to comment specifically on that.